### PR TITLE
feat: debug http response

### PR DESCRIPTION
**Pull request by kindermoumoute**

### CODE REVIEW

File: scw/client.go [hunk 1]
```diff
@@ -198,6 +198,14 @@ func (c *Client) do(req *ScalewayRequest, res interface{}) (err error) {
 			err = closeErr
 		}
 	}()
+	if logger.ShouldLog(logger.LogLevelDebug) {
+		dump, err := httputil.DumpResponse(resp, true)
```
Comment (DenBeke) [inline scw/client.go]: Can this contain sensitive data?

Comment (jerome-quere) [reply scw/client.go]: in response hopfully not.

Comment (kindermoumoute) [reply scw/client.go]: Nope

Comment (DenBeke) [reply scw/client.go]: 👍

**Pull request merged**